common-auth/SECURITY.md

```diff
@@ -0,0 +1,65 @@
+# Running registry and streamline web-services securely
+
+This module is intended to be used by registry and streamline web-services so that they can enable http client authentication via SPNEGO. Currently supported 
```

Comment:
@hmcl I did not get the part about any registry dependent web-services. Should we really add that?